doc/devel/contributing.rst

```diff
@@ -437,6 +437,73 @@ forced to use ``**kwargs``.  An example is
       elif len(args) == 1:
          ...etc...
 
+.. _using_logging:
+
+Using logging for debug messages
+--------------------------------
+
+Matplotlib uses the standard python logging_ library to write verbose
```

Comment:
We have an intersphinx mapping (https://github.com/matplotlib/matplotlib/blob/master/doc/conf.py#L95) to the stdlib so you can just write
```
`logging`
`print`
`warnings.warn`
```
etc. and they will link to the corresponding entries in the stdlib docs